wp-content/themes/core/assets/pcss/_variables.pcss

```diff
@@ -0,0 +1,17 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Variables
+ *
+ * This file is just a clearing-house.
+ * Make partials (start with an underscore) elsewhere for actual code.
+ *
+ * ----------------------------------------------------------------------------- */
+
+/* Colors */
+@import "global/color/_variables.pcss";
+
+/* Custom Selectors */
+@import "global/custom-selectors/_variables.pcss";
+
+/* Media Queries */
+@import "global/media-queries/_variables.pcss";
```

Comment:
Using the global imports plugin eliminates the need for us to do this, and it provides access to all these partials when we do block-specific entrypoints. :)

Reply:
✔️ This file is gone now